src/main/java/pro/cloudnode/smp/bankaccounts/POS.java

```diff
@@ -305,6 +309,14 @@ public static void openOwnerGui(final @NotNull Player player, final @NotNull Che
         ));
         gui.addItem(items);
 
+        // clear bottom row if the chest is full (#35)
+        if (extraRows == -1) {
+            for (int i = size - 9; i < size; i++) {
+                gui.clear(i);
+            }
+        }
```

Comment:
imo should just add/remove pager button instead of removing the entire row

Reply:
well what this does is basically before this the entire 6 rows are filled with the items, so it just clears the last row for the actual UI buttons (see fig 1)

![fig 1. ui panel, confirm, info, and decline button](https://github.com/cloudnode-pro/BankAccounts/assets/50217798/e67fcf84-7c4f-4cda-ae42-b3e12549f6d2)

Reply:
as changed in https://github.com/cloudnode-pro/BankAccounts/pull/41/commits/3ef2b143c8c979ebf98221dbaf83e688bb3fe95e , still got to clear that row because otherwise the items will overflow into the bar.

![image](https://github.com/cloudnode-pro/BankAccounts/assets/50217798/4323440c-2d9c-47ff-9859-fd14fb5c7d75)

Reply:
> as changed in [3ef2b14](https://github.com/cloudnode-pro/BankAccounts/commit/3ef2b143c8c979ebf98221dbaf83e688bb3fe95e) , still got to clear that row because otherwise the items will overflow into the bar.
> 
> ![image](https://user-images.githubusercontent.com/50217798/263508390-4323440c-2d9c-47ff-9859-fd14fb5c7d75.png)

should just limit the `for` loop that puts the items to never reach the last row

Reply:
well we just inventory#add(items) and that adds all of them, no for loop required

Reply:
> well we just inventory#add(items) and that adds all of them, no for loop required

maybe we should for loop instead then so we dont have to add items then remove some and add items again

Reply:
did so in c5ff919